org.thecrossingchurch.CustomJobs/Jobs/HubspotIntegrationMatching.cs

```diff
@@ -0,0 +1,718 @@
+            Debug.WriteLine( string.Format( "Comparing {0} Top-Level Domains.", tlds.Length ) );
+
+            // Get HS API Token
+            string attrKey = dataMap.GetString("AttributeKey");
+            string attrValue = GlobalAttributesCache.Get().GetValue(attrKey);
+            key = Encryption.DecryptString(attrValue);
+
+            // Don't see this being used for anything...
+            var current_id = 0;
```

Comment:
This doesn't seem to ever be incremented, so it is always 0. This defeats the purpose of having this number. Can we either add code to increment it or remove it entirely?